applications/omnipresence/src/omnipresence_listener.erl

```diff
@@ -122,16 +128,38 @@ handle_cast('find_subscriptions_srv', #state{subs_pid=_Pid}=State) ->
             {'noreply', State#state{subs_pid='undefined'}};
         P when is_pid(P) ->
             lager:debug("new subs pid: ~p", [P]),
+            gen_listener:cast(self(), 'send_sync'),
             {'noreply', State#state{subs_pid=P
                                     ,subs_ref=erlang:monitor('process', P)
                                    }}
     end;
-handle_cast({'gen_listener',{'created_queue',_Queue}}, State) ->
+handle_cast({'gen_listener',{'created_queue',Queue}}, State) ->
+    gen_listener:cast(self(), 'send_sync'),
+    {'noreply', State#state{queue=Queue}};
+handle_cast({'gen_listener',{'is_consuming',IsConsuming}}, State) ->
+    gen_listener:cast(self(), 'send_sync'),
+    {'noreply', State#state{consuming=IsConsuming}};
+handle_cast('send_sync', #state{subs_pid=Pid, queue=Queue, consuming=IsConsuming} = State)
+  when Pid =:= 'undefined' 
+  orelse Queue =:= 'undefined' 
+  orelse IsConsuming =:= 'false'  ->
+    {'noreply', State};
+handle_cast('send_sync', #state{subs_pid='undefined'}=State) ->
+    {'noreply', State};
+handle_cast('send_sync', #state{queue='undefined'}=State) ->
     {'noreply', State};
-handle_cast({'gen_listener',{'is_consuming',_IsConsuming}}, State) ->
+handle_cast('send_sync', #state{consuming='false'}=State) ->
     {'noreply', State};
+handle_cast('send_sync', #state{subs_pid=Pid, queue=Queue, consuming='true', sync='false'} = State) ->
+    Payload = wh_json:from_list(
+                [{<<"Action">>, <<"Request">>}
```

Comment:
I see handlers for Action = "Start" and "End" but not "Request" in omnip_subscriptions